charts/kubefed/values.yaml

```diff
@@ -49,7 +49,7 @@ controllermanager:
     annotations: {}
     podAnnotations: {}
     replicaCount: 2
-    repository: quay.io/kubernetes-multicluster
+    repository: docker.io/mesosphere
```

Comment:
We could use ghcr.io instead as this is a public project :thinking:

Reply:
fixed in ed934d38c3ecdaf2fb2be38334b7dbc52b7c1e83,  i ran `make push` locally and it failed with (as expected) : 

```
...
 => [8/8] RUN chown -R hyperfed:hyperfed /hyperfed                                                                                                                                               0.5s
 => exporting to image                                                                                                                                                                           0.3s
 => => exporting layers                                                                                                                                                                          0.3s
 => => writing image sha256:7dde47dfd60d9edf0e71e4b48915d786ba09816c50958667878c0bf3a4d3021e                                                                                                     0.0s
 => => naming to ghcr.io/mesosphere/kubefed:v0.9.1-61-ged934d38-dirty  
rm -f images/kubefed/hyperfed
docker push ghcr.io/mesosphere/kubefed:v0.9.1-61-ged934d38-dirty
The push refers to repository [ghcr.io/mesosphere/kubefed]
99284a7e801b: Preparing 
6b8ee1bf7d13: Preparing 
83f57be68129: Preparing 
5f70bf18a086: Preparing 
9783020602e4: Preparing 
c43227f5b8cd: Waiting 
030fd6593b51: Waiting 
7cd52847ad77: Waiting 
unauthorized: unauthenticated: User cannot be authenticated with the token provided.
make: *** [Makefile:141: push] Error 1
```

Reply:
I'm sorting this with devprod, and have updated workflows permissions to be able to push to this image already. The image is publicly available. I need to also link the code repository to ensure it's free storage, but waiting on devprod to give me  admin permissions on the repo.